nntrainer/src/conv2d_layer.cpp

```diff
@@ -370,6 +493,108 @@ int Conv2DLayer::conv2d(float *in, TensorDim indim, const float *kernel,
     }
     I++;
   }
+  return status;
+}
+
+int Conv2DLayer::conv2d_gemm(const float *mKernel, TensorDim kdim,
```

Comment:
I think `in` does not need to be tensor rather float * as stated above

Reply:
needed to calculate padding.

Reply:
If that's the case, I guess padding can be done outside of conv2d to avoid memcpy. I think it could be done later.